DevExtreme.AspNet.TagHelpers.Generator/ChartSeriesSpecificSettings/Entry.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Xml.Serialization;
+
+namespace DevExtreme.AspNet.TagHelpers.Generator.ChartSeriesSpecificSettings {
```

Comment:
I suggest reverting "PropOf" back to `Xml.Linq`. Cannot see any benefits in using `XmlSerializer` for it.